addons/mail/controllers/thread.py

```diff
@@ -92,6 +92,9 @@ def mail_thread_recipients_get_suggested_recipients(self, thread_model, thread_i
         thread = self._get_thread_with_access(thread_model, thread_id)
         partner_ids = request.env['res.partner'].search([('id', 'in', partner_ids)])
         recipients = thread._message_get_suggested_recipients(reply_discussion=True, additional_partners=partner_ids, primary_email=main_email)
+        if partner_ids:
```

Comment:
So globally, partner_ids is more like "new_customer_ids" ?  Worth a docstring, and we can change the parameter name in master :)